core/server/src/test/java/com/google/common/util/concurrent/FakeRateLimiter.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package com.google.common.util.concurrent;
+
+import static java.util.concurrent.TimeUnit.MICROSECONDS;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+import com.google.common.collect.Lists;
+import com.google.common.util.concurrent.RateLimiter.SleepingTicker;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Provides a fake {@link RateLimiter} to facilitate testing. Note that in Guava 14.0 the
+ * {@link RateLimiter$SleepingTicker} class is package private, so this fake rate limiter has to
+ * be in the same package to be able to extend it. Later version of Guava makes it public, so this
+ * can be moved to a package belongs to alluxio when updating to a newer version of Guava.
+ */
+public class FakeRateLimiter {
```

Comment:
`FakeRateLimiter` or `MockRateLimiter`? when you say fake, I thought the class is not working, is it accurate?